docs/UserGuide.md

```diff
@@ -100,8 +104,8 @@ Format: `findi KEYWORD [MORE_KEYWORDS]`
   e.g. `shirt` will return `dress shirt`, `collared shirt`
 
 Examples:
-- `findi oil` returns `Olive Oil` and `Vegetable Oil`
-- `findi blue` returns `Blue Shirt`, `Blue Pants`
+- `findi oil` returns orders with the keyword `oil` such as `Olive Oil` and `Vegetable Oil`
+- `findi yellow pillow` returns orders with the keywords `yellow` and `pillow` such as `yellow blanket`, `ergonomic pillow` and `yellow pillow`
```

Comment:
I think you meant something like `returns items with item names containing `oil` such as ...` here. Do change it for all relevant sections